content/enterprise_influxdb/v1/about-the-project/release-notes.md

```diff
@@ -21,15 +21,58 @@ InfluxDB Enterprise builds are available. For more information, see
 [FIPS-compliant InfluxDB Enterprise builds](/enterprise_influxdb/v1/introduction/installation/fips-compliant/).
 {{% /note %}}
 
+{{% note %}}
+#### Series file compaction
+
+With InfluxDB Enterprise v1.11.4+, InfluxDB can be configured to optionally
+run the `influxd_inspect -compact-series-file` command to [compact series files](/enterprise_influxdb/v1/tools/influx_inspect/#--compact-series-file-) before data nodes are started.
```

Comment:
```suggestion
run the `influxd_inspect compact-series-file` command to [compact series files](/enterprise_influxdb/v1/tools/influx_inspect/#compact-series-file) before data nodes are started.
```

1. Is the command supposed to have this leading dash?
2. Do we need to add the entry for it in `/enterprise_influxdb/v1/tools/influx_inspect/` before publishing?

Reply:
All the v1 CLI's use single-leading-dash flags. The flag is [already documented](https://docs.influxdata.com/enterprise_influxdb/v1/tools/influx_inspect/#--compact-series-file-) and is formatted correctly because of the old flag style used in these docs (by a previous technical writer). It went out as part of Enterprise 1.11.3. @gunnaraasen The change here is that this is no longer the default?